config/custom_schema_types.config

```diff
@@ -134,6 +151,7 @@ custom_schema_types {
     types = [
         'InputNamespace': custom_schema_types.check_input_namespace,
         'BAMEntryList': custom_schema_types.check_bam_list,
+        'VCFEntryList': custom_schema_types.check_vcf_list,
```

Comment:
One more thing, because intersection should be done on at least 2 VCFs, it would be worth adding a check that at least 2 VCFs are given when running with that type

Reply:
Check added, and I tested with 1,2 and 3 input VCFs.  All worked as expected.